src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/ActionGroupsApiTest.java

```diff
@@ -174,17 +174,20 @@ public void testActionGroupsApi() throws Exception {
         Assert.assertFalse(response.getBody().contains("Resource 'GET_UT' is read-only."));
 
         // -- GET_UT hidden resource, must be 404
-        rh.sendAdminCertificate = true;
+        rh.sendAdminCertificate = false;
```

Comment:
Please add a super admin test case that checks that the resource is visible and marked as hidden.

Reply:
I didnt wanted it it be false as I have separate test cases for it.

I should have reverted it, but I guess, i missed out. Reverting now